libs/application/templates/children-residence-change/src/fields/ContactInfo/index.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { Controller, useFormContext } from 'react-hook-form'
+import { InputController } from '@island.is/shared/form-fields'
+import {
+  Box,
+  GridColumn,
+  GridContainer,
+  GridRow,
+  Icon,
+  Input,
+  Text,
+} from '@island.is/island-ui/core'
+import { contactInfo } from '../../lib/messages'
+import { childrenResidenceInfo } from '../../lib/utils'
+import { CRCFieldBaseProps } from '../../types'
+
+const emailId = 'parentA.email'
+const phoneNumberId = 'parentA.phoneNumber'
+
+const counterPartyEmail = 'counterParty.email'
+const counterPartyPhoneNumber = 'counterParty.phoneNumber'
+
+export const contactInfoIds = [
+  emailId,
+  phoneNumberId,
+  counterPartyEmail,
+  counterPartyPhoneNumber,
+]
+
+const ContactInfo = ({ errors, application }: CRCFieldBaseProps) => {
+  const { answers, externalData } = application
+  const { userProfile, nationalRegistry } = externalData
+  const { formatMessage } = useIntl()
+  const { clearErrors } = useFormContext()
+  const emailError = errors?.parentA?.email
+  const phoneNumberError = errors?.parentA?.phoneNumber
+  const counterPartyError = errors?.counterParty as string
+  const isCounterPartyError = typeof counterPartyError === 'string'
+  const counterPartyEmailError = errors?.counterParty?.email
+  const counterPartyPhoneError = errors?.counterParty?.phoneNumber
+  const applicant = nationalRegistry.data
+  const childResidenceInfo = childrenResidenceInfo(applicant, answers)
+
+  return (
+    <>
+      <Text marginTop={3}>
```

Comment:
I might refactor and extract some of this to a seperate component this when working on this task: https://app.asana.com/0/1199733748699731/1200222247492226